ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -1828,8 +1815,11 @@ def flatten(*, backend: str, ret):
     return ret_flat
 
 
-def flatten_frontend(*, ret, backend: str, frontend_array_fn=None):
+def flatten_frontend(*, ret, backend: str, frontend_array_fn=None, frontend_config=None):
```

Comment:
Is the `frontend_config` required?

Reply:
It's to handle scalars in case there is no `frontend_array_fn`.

Reply:
No actually. I have removed it.